src/callbacks/callbacks.jl

```diff
@@ -61,6 +63,12 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
     (; idealized_insolation, idealized_h2o, idealized_clouds) = p.radiation
     (; ᶠradiation_flux, radiation_model) = p.radiation
 
+    # If we have prescribed aerosols, we need to update them
+    for (key, tv) in pairs(p.tracers.prescribed_aerosol_timevaryinginputs)
```

Comment:
This is very elegant. It's awesome how all the details are hidden in utilities. 

At some point (once we have a 2-moment scheme) we might want to put the aerosol loading in a separate place, to ensure it's happening before both the radiation and microphysics. But that will probably not happen in the next couple of months.